release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,195 +1,223 @@
-* Open v1.11 PRs
-   * `is:open is:pr milestone:v1.11 -label:"kind/feature" -label:"kind/failing-test"` [milestone open PRs, excluding kind/feature and kind/failing-test](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+milestone%3Av1.11+-label%3A%22kind%2Ffeature%22+-label%3A%22kind%2Ffailing-test%22+)
+One of the main purposes of Code Freeze is to get issues and PRs filtered through the use of labels, with `milestone` and the `priority` label playing an important role on that.
 
-### Reports
+The following items help with achieving the above:
 
-No reports are required during this period, although you should be maintaining a red/yellow/green report template starting now.
+* issues targeting your release should have a *milestone*, kind, and sig.
+* PRs linked to these issues should have the same labels and milestone.
 
-## Burndown 
+If the needed additions on the issues/PRs do not match the above, you can add the relevant labels.
 
-At the beginning of Burndown, all issues to stay in the milestone need to have the following characteristics.  PRs for the milestone should share the same labels.
+If you are not sure (mainly priority), you can comment and ask the owner or SIG leads about the status of the issue/PR (also see [Escalation Path](#escalation-path) on pinging options).
 
-* kind, sig, and milestone labels
+Other regular patterns are:
 
-All bugs should also show progress towards resolution and that they're getting attention.  If they're not, you need to get the attention of the SIG(s) on those specific bugs, and find out if they're going to fix them or target the fix for a future milestone instead.  Also, SIGs need to make a decisions on milestone inclusion and prioritization; you need to remind them to do so on each bug for example reminding them to run the "/milestone vX.YY" and "/priority ${prioritylevel}" Prow commands.
+Stuck PRs: Sometimes PRs get stuck in the approval process, as every PR requires both a `/lgtm` and an `/approve` by an Owner. When you detect such cases, you should remind relevant SIG Leads / Owners to review these.
 
-Even when bugs have PRs resolving them, these PRs can get stuck in the approval process. This means it's your job to remind SIG leads of any stuck PRs until they get approved and merged.
+CI Signal: Checking newly-reported test failures becomes more important; you should assume that any new failure is related to the upcoming release, bring to the attention of the CI Signal lead, and assist in getting follow-up from the appropriate SIG.  Generally the CI Signal lead will track these failures, but newly opened issues may not initially have sufficient labelling to catch their attention.
```

Comment:
bold/italicize "CI Signal"

Reply:
also a bit of a run-on sentence: how about -
"Checking newly-reported test failures becomes more important; you should assume that any new failure is related to the upcoming release. Bring the issue to the attention of the CI Signal lead, and assist in getting follow-up from the appropriate SIG. [etc]"

Reply:
good catch